tree/ntuple/v7/test/ntuple_storage_daos.cxx

```diff
@@ -299,4 +299,46 @@ TEST_F(RPageStorageDaos, CagedPages)
       }
    }
 }
+
+TEST_F(RPageStorageDaos, Checksum)
+{
+   std::string daosUri = RegisterLabel("ntuple-test-checksum");
+   CreateCorruptedRNTuple(daosUri);
+
+   {
+      IMTRAII _;
+
+      auto reader = RNTupleReader::Open("ntpl", daosUri);
+      EXPECT_EQ(1u, reader->GetNEntries());
+
+      auto viewPx = reader->GetView<float>("px");
+      auto viewPy = reader->GetView<float>("py");
+      EXPECT_THROW(viewPy(0), RException); // we run under IMT, even the valid column should fail
+   }
```

Comment:
I'm surprised this works even if IMT is not compiled in...

Reply:
This test is in an `#ifdef R__USE_IMT` block.

Reply:
Ah, it was already there. I missed that...